Lib/idlelib/configdialog.py

```diff
@@ -1503,8 +1517,6 @@ def create_page_font_tab(self):
         indent_title.pack(side=TOP, anchor=W, padx=5)
         self.indent_scale.pack(side=TOP, padx=5, fill=X)
 
-        return frame
```

Comment:
No, because calling a class returns a new instance.  In this case, the instance, self, is a frame.  Create_widgets is now called within __init__ (and the None return ignored) instead of by ConfigDialog.